lib/Notes/components/NotesAccordion/NotesAccordion.js

```diff
@@ -71,6 +71,7 @@ export default class NotesAccordion extends Component {
         {!hideAssignButton && (
           <IfPermission perm="ui-notes.item.assign-unassign">
             <Button
+              id="note-assign"
```

Comment:
Since you established the `...-button` convention above, it would be great to follow through with it. Consistency makes test-writing so much easier.